week1/armstrong.rb

```diff
@@ -9,5 +9,23 @@
 # and `false` otherwise.
 
 def armstrong_number(n)
-    raise NotImplementedError # Delete this line and start coding here
+    puts "Enter a number"
+    n=gets.chomp.to_i
+
+    temp=n
+    sum = 0
+
+    while n!=0
+        rem=n%10
+        n=n/10
+        sum=sum+rem*rem*rem
+    end
```

Comment:
You can easily find the digits using:
```ruby
array_of_digits = n.digits
```
And power can be easily done using `**` operator